kura/org.eclipse.kura.core.cloud/src/main/java/org/eclipse/kura/core/cloud/MessageHandlerCallable.java

```diff
@@ -236,11 +236,11 @@ public KuraMessage setResponseCode(KuraPayload payload, int responseCode) {
     public void setException(KuraPayload payload, Throwable t) {
         if (t != null) {
             payload.addMetric(METRIC_EXCEPTION_MSG, t.getMessage());
-            payload.addMetric(METRIC_EXCEPTION_STACK, stackTraceAsString(t));
         }
     }
 
-    private String stackTraceAsString(Throwable t) {
+    @SuppressWarnings("unused")
+	private String stackTraceAsString(Throwable t) {
```

Comment:
Please check the formatting. As a reference, please use the cleanup and formatting profiles available here: https://github.com/eclipse/kura/tree/develop/kura/setups/formatting